packages/components/src/components/tab-group/tab-group.stories.ts

```diff
@@ -54,317 +64,35 @@ export const Default = {
+ * Use the `activation` attribute to define how the tab panel is shown when the user interacts with it.
+ *
+ * - `auto`: Automatically shows the corresponding tab panel.
+ * - `manual`: Requires user interaction to show the corresponding tab panel.
  */
-
-export const SampleDeepLink = {
-  name: 'Sample: Deep Link',
-  render: () => {
-    return html`
-      <sd-tab-group id="deep-link-tab-group">
-        <sd-tab slot="nav" panel="tab-1">Tab 1</sd-tab>
-        <sd-tab slot="nav" panel="tab-2">Tab 2</sd-tab>
-        <sd-tab slot="nav" panel="tab-3">Tab 3</sd-tab>
-        <sd-tab slot="nav" panel="tab-4">Tab 4</sd-tab>
-        <sd-tab slot="nav" panel="tab-5">Tab 5</sd-tab>
-
-        <sd-tab-panel name="tab-1"><div class="slot slot--text slot--border">Tab panel 1</div></sd-tab-panel>
-        <sd-tab-panel name="tab-2"><div class="slot slot--text slot--border">Tab panel 2</div></sd-tab-panel>
-        <sd-tab-panel name="tab-3"><div class="slot slot--text slot--border">Tab panel 3</div></sd-tab-panel>
-        <sd-tab-panel name="tab-4"><div class="slot slot--text slot--border">Tab panel 4</div></sd-tab-panel>
-        <sd-tab-panel name="tab-5"><div class="slot slot--text slot--border">Tab panel 5</div></sd-tab-panel>
-      </sd-tab-group>
-
-      <sd-button id="deep-link-btn"> Open Panel 5 </sd-button>
-
-      <script type="module">
-        const btn = document.getElementById('deep-link-btn');
-        const sdTabGroup = document.getElementById('deep-link-tab-group');
-
-        const nextActivePanel = document.querySelector('sd-tab[panel="tab-5"]');
-
-        btn.addEventListener('click', () => {
-          sdTabGroup.setActiveTab(nextActivePanel);
-        });
-      </script>
-    `;
-  }
+export const Activation = {
+  name: 'Activation',
+  render: () => html`
+    <sd-tab-group activation="manual">
+      <sd-tab slot="nav" panel="tab-1" variant="default">Tab 1</sd-tab>
+      <sd-tab-panel name="tab-1">
+        <div>Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.</div>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-2" variant="default">Tab 2</sd-tab>
+      <sd-tab-panel name="tab-2">
+        <div>Provident illo neque vel ex.</div>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-3" variant="default">Tab 3</sd-tab>
+      <sd-tab-panel name="tab-3">
+        <div>Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.</div>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-4" variant="default">Tab 4</sd-tab>
+      <sd-tab-panel name="tab-4">
+        <div>Inventore perspiciatis delectus nisi doloremque soluta inventore.</div>
+      </sd-tab-panel>
+      <sd-tab slot="nav" panel="tab-5" variant="default">Tab 5</sd-tab>
+      <sd-tab-panel name="tab-5">
+        <div>Provident nisi doloremque soluta inventore.</div>
+      </sd-tab-panel>
+    </sd-tab-group>
```

Comment:
@smfonseca I would expect to see the "auto" version here as well. Only the manual tab-group is showcased in the story, which I feel won't be enough to showcase their differences.
<img width="1099" alt="image" src="https://github.com/user-attachments/assets/ed67a819-138b-404d-a8fd-ffca1ff01b81">